src/main/java/meo/MeoException.java

```diff
@@ -0,0 +1,13 @@
+package meo;
+public class MeoException extends Exception{
+    public static String taskMissing = "Meow, please insert your task.";
+    public static String deadlineTime = "Meow, please include your deadline with /by.";
+    public static String eventTime = "Meow, please include your from and to timing with /from and /to.";
```

Comment:
Perhaps a more intuitive variable name here?
```suggestion
    public static String invalideventTime = "Meow, please include your from and to timing with /from and /to.";
```